rsocket-core/src/main/java/io/rsocket/buffer/Tuple2ByteBuf.java

```diff
@@ -0,0 +1,415 @@
+package io.rsocket.buffer;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufAllocator;
+import io.netty.buffer.Unpooled;
+import io.netty.util.ReferenceCountUtil;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.ByteBuffer;
+import java.nio.channels.FileChannel;
+import java.nio.channels.GatheringByteChannel;
+import java.nio.charset.Charset;
+import org.agrona.BufferUtil;
+
+class Tuple2ByteBuf extends AbstractTupleByteBuf {
+  private static final int MEMORY_CACHE_ALIGNMENT = 64;
```

Comment:
this should be changed to 8

Reply:
fixed. Now I'm using native Netty property `SystemPropertyUtil.getInt("io.netty.allocator.directMemoryCacheAlignment", 0);`